src/vue-router/index.js

```diff
@@ -0,0 +1,7 @@
+export const vueRoutes = Object.freeze({
```

Comment:
let’s bind somehow these constants with vue-router

Reply:
resolved and moved to backlog

https://app.asana.com/0/894507180698157/950061174802141/f

will be a little bit easier to complete after removing legacy stuff